README.md

```diff
@@ -11,6 +11,42 @@
 
 This package allows us to reuse GitHub actions, which simplifies version management.
 
+### Example of use of the action [Composer require checker](https://github.com/maglnet/ComposerRequireChecker)
```

Comment:
```suggestion
### Using [Composer require checker](https://github.com/maglnet/ComposerRequireChecker)
```

Reply:
(Nitpick) Maybe change other headers too to be consistent.